scenarios/kubernetes_e2e.py

```diff
@@ -689,7 +717,12 @@ def create_parser():
         default=[],
         help='Send unrecognized args directly to kubetest')
 
-    # aws
+    # kops & aws
+    # At the moment, the logic is that `--aws` => kops & aws,
+    # and `--kops` => kops & gce.
+    # TODO(justinsb): get this using a `--provider` flag, just like kubetest
+    parser.add_argument(
+        '--kops', action='store_true', help='E2E job runs using kops')
```

Comment:
`--kops-gce`? `--kops` is somewhat confusing, probably just change it to `--provider`, and pass through to kubetest